tools/peptideshaker/searchgui.xml

```diff
@@ -1,4 +1,4 @@
-<tool id="search_gui" name="Search GUI" version="1.28.0">
+<tool id="search_gui" name="Search GUI" version="2.1.1">
```

Comment:
Can we reuse the version macro here. Ideally we don't need to update the version number manually.